platform/security/sts/security-sts-pkivalidator/src/main/java/org/codice/ddf/security/validator/pki/PKITokenValidator.java

```diff
@@ -204,10 +208,40 @@ public TokenValidatorResponse validateToken(TokenValidatorParameters tokenParame
       Credential returnedCredential = validator.validate(credential, requestData);
       X500Principal subjectX500Principal =
           returnedCredential.getCertificates()[0].getSubjectX500Principal();
+
       response.setPrincipal(subjectX500Principal);
       if (response.getAdditionalProperties() == null) {
         response.setAdditionalProperties(new HashMap<>());
       }
+
+      try {
+        if (returnedCredential.getCertificates() != null) {
+          Collection<List<?>> subjectAlternativeNames =
+              returnedCredential.getCertificates()[0].getSubjectAlternativeNames();
```

Comment:
Can `returnedCredential.getCertificates()` be empty?

Reply:
Removing this check as the line previous already has accessed the certificates at this point.